### DISCUSSION TITLE
I have been applied multiple time with my college id card but it rejecting always

### DISCUSSION THREAD

**Discussion by Omkar-bhutale** [community/community#138793] [GitHub Education]
### Hi everyone,

<html>
<body>

Hey Bhutale Omkar Pavankumar,Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Sep 16, 2024 for @Omkar-bhutale.Please use your device camera to submit your academic affiliation documents. If you’re having trouble using your device camera, please allow camera access for the application website on your browser. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found here. You may also try applying from a different computer or mobile device.Once you have resolved the issues with your sumbission, visit us again and reapply.Have questions? Please seek help via our Student Developer Pack Application FAQ or visit the GitHub Education Community. The GitHub Education Team
--

</body>
</html>Hey Bhutale Omkar Pavankumar,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Sep 16, 2024 for [@Omkar-bhutale](https://github.com/Omkar-bhutale).

Please use your device camera to submit your academic affiliation documents. If you’re having trouble using your device camera, please allow camera access for the application website on your browser. The steps for this will vary depending on which browser you are using. Information on browsers supported by GitHub can be found [here](https://docs.github.com/en/get-started/using-github/supported-browsers). You may also try applying from a different computer or mobile device.

Once you have resolved the issues with your sumbission, [visit us again and reapply](https://education.github.com/).

Have questions? Please seek help via our [Student Developer Pack Application FAQ ](https://github.com/orgs/community/discussions/111352)or visit the [GitHub Education Community.](https://github.com/orgs/community/discussions/categories/github-education)

💖 The GitHub Education Team
![image](https://github.com/user-attachments/assets/a16606bc-0bbe-4475-829d-b0d9f19d489b)

**Comment by Omkar-bhutale** [comment]
I had used both my laptop integrated camera and mobile camera

**Comment by Omkar-bhutale** [comment]
my id card 
![IDCARD-Photoroom](https://github.com/user-attachments/assets/7b1c7a78-ffe9-4f6a-9f08-f846d1d8be77)

**Comment by QW60** [comment]
Hi! You could try to send in your class schedule together with your ID, Also make sure that your billing information is the same as on your student ID, you don't have to add a payment method but the info should match. I would personally also recommend sending your ID and other images without a removed background as that may appear more legit.

**Reply by Omkar-bhutale** [reply]
how can i am not getting an option of uploding it only via camera from laptop cam a can not ,if i try using mobile only front camera not back camera working

**Reply by QW60** [reply]
For verification reasons you sometimes are not allowed to upload, this is so student ID's are not forged. I recommend trying with either your phone or laptop to make a picture of your ID and your Schedule or other proof of enrollment like official documents with your name on.
